File: sample-basic-java/src/main/java/org/dbtools/sample/model/database/main/individualdata/IndividualData.java
/*
 * IndividualData.java
 *
 * Created: 11/20/2016 10:57:23
 */



package org.dbtools.sample.model.database.main.individualdata;

import android.database.Cursor;
import org.dbtools.android.domain.database.contentvalues.DBToolsContentValues;


public class IndividualData extends IndividualDataBaseRecord {


    public IndividualData(IndividualData record) {
        super(record);
    }

    public IndividualData(Cursor cursor) {
        setContent(cursor);
    }

    public IndividualData(DBToolsContentValues values) {
        setContent(values);
    }

    public IndividualData() {
    }


}

File: sample-basic-kotlin/src/main/kotlin/org/dbtools/sample/kotlin/model/database/main/individualdata/IndividualData.kt
/*
 * IndividualData.kt
 *
 * Created: 11/20/2016 11:05:51
 */



package org.dbtools.sample.kotlin.model.database.main.individualdata

import android.database.Cursor
import org.dbtools.android.domain.database.contentvalues.DBToolsContentValues


class IndividualData : IndividualDataBaseRecord {


    constructor() {
    }

    constructor(record: IndividualData) : super(record) {
    }

    constructor(cursor: Cursor) {
        setContent(cursor)
    }

    constructor(values: DBToolsContentValues<*>) {
        setContent(values)
    }


}